lib/yardstick/parser.rb

```diff
@@ -10,7 +10,7 @@ class Parser
     #
     # @api private
     def self.parse_paths(paths)
-      YARD.parse(paths, [], YARD::Logger::ERROR)
+      YARD.parse(paths, [], Logger::ERROR)
```

Comment:
This probably passes in `::Logger::ERROR` directly. I'm fine if it continues to work, I just think it's interesting that it does.